cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/components/total-purchase-amount.js

```diff
@@ -0,0 +1,28 @@
+export const TotalPurchaseAmount = () => {
+  return (
+    <div className="m-tool-card__bottom-border">
+      <form>
+        <div className=" m-tool__tag">step 1/4</div>
+        <h2>Total purchase amount</h2>
+        <div className="one-line-desc">
+          Estimate the cost of the item including tax.
+        </div>
+
+        <div class="m-form-field-with-button">
```

Comment:
`className` on this line and the one below it

Reply:
completed